angular-workspace/projects/ni/nimble-angular/pipes/format-number-decimal.pipe.ts

```diff
@@ -0,0 +1,65 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { DecimalUnitFormat } from '@ni/nimble-components/dist/esm/utilities/unit-format/decimal-unit-format';
+import { passthroughUnitScale } from '@ni/nimble-components/dist/esm/utilities/unit-format/unit-scale/passthrough-unit-scale';
+import { byteUnitScale } from '@ni/nimble-components/dist/esm/utilities/unit-format/unit-scale/byte-unit-scale';
+import { byte1024UnitScale } from '@ni/nimble-components/dist/esm/utilities/unit-format/unit-scale/byte-1024-unit-scale';
+import { voltUnitScale } from '@ni/nimble-components/dist/esm/utilities/unit-format/unit-scale/volt-unit-scale';
+import type { UnitScale } from '@ni/nimble-components/dist/esm/utilities/unit-format/unit-scale/unit-scale';
+
+export { byteUnitScale, byte1024UnitScale, voltUnitScale };
```

Comment:
I don't like the idea of exporting our unit scales from both the number format pipes, but making a separate module just for them seems weird, too. Open to suggestions.

Reply:
I lean towards creating a separate module for them so that we only import them if they are being used, but I'd be interested in hearing if other code owners have opinions before moving things around too much.